Michelf/Markdown.inc.php

```diff
@@ -1,10 +1,16 @@
 <?php
-
-# Use this file if you cannot use class autoloading. It will include all the
-# files needed for the Markdown parser.
-#
-# Take a look at the PSR-0-compatible class autoloading implementation
-# in the Readme.php file if you want a simple autoloader setup.
+/**
+ * Use this file if you cannot use class autoloading. It will include all the
+ * files needed for the Markdown parser.
+ *
+ * Take a look at the PSR-0-compatible class autoloading implementation
+ * in the Readme.php file if you want a simple autoloader setup.
+ *
+ * @package   php-markdown
+ * @author    Michael Fortin <michael.fortin@michaelf.com>
```

Comment:
Sorry! My mistake!

On Friday, 4 March 2016, Michel Fortin <notifications@github.com> wrote:

> In Michelf/Markdown.inc.php
> <https://github.com/michelf/php-markdown/pull/229#discussion_r54891650>:
>
> > @@ -1,10 +1,16 @@
> >  <?php
> > -
> > -# Use this file if you cannot use class autoloading. It will include all the
> > -# files needed for the Markdown parser.
> > -#
> > -# Take a look at the PSR-0-compatible class autoloading implementation
> > -# in the Readme.php file if you want a simple autoloader setup.
> > +/**
> > + * Use this file if you cannot use class autoloading. It will include all the
> > + * files needed for the Markdown parser.
> > + *
> > + * Take a look at the PSR-0-compatible class autoloading implementation
> > + * in the Readme.php file if you want a simple autoloader setup.
> > + *
> > + * @package   php-markdown
> > + * @author    Michael Fortin <michael.fortin@michaelf.com <javascript:_e(%7B%7D,'cvml','michael.fortin@michaelf.com');>>
>
> That should actually be Michel Fortin <michel.fortin@michelf.com
> <javascript:_e(%7B%7D,'cvml','michel.fortin@michelf.com');>>. There's no
> "a" in my name.
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/michelf/php-markdown/pull/229/files#r54891650>.
>


-- 
*Robbie Averill*
LA PETITE MANOUCHE - NZ Gypsy Jazz Guitar Duo
Phone: +64 27 321 4758
contact@lapetitemanouche.com
www.lapetitemanouche.com